src/frontend/src/lib/components/signer/SignerOrigin.svelte

```diff
@@ -34,7 +34,7 @@
 {#if nonNullish(origin)}
 	<p class="mb-6 break-normal text-center">
 		{$i18n.signer.origin.text.request_from}
-		{#if nonNullish(host)}<span class="font-bold text-brandeis-blue"
+		{#if nonNullish(host)}<span class="font-bold text-foreground-brand-primary"
```

Comment:
This information is particularly important. Can you provide a screenshot - i.e. have you double checked it is visually still predominent?

Reply:
This is the screenshot for that specific one

![Screenshot 2024-11-01 at 08 32 58](https://github.com/user-attachments/assets/576dd11a-62b3-4d15-b356-414f9d8577c8)

Reply:
Thx!